src/libs/algolia/fetchAlgolia/fetchSearchResults/fetchSearchResults.ts

```diff
@@ -63,6 +66,7 @@ export const fetchSearchResults = async ({
         /* Is needed to get a queryID, in order to send analytics events
          https://www.algolia.com/doc/api-reference/api-parameters/clickAnalytics/ */
         clickAnalytics: true,
+        ...(aroundPrecision && aroundPrecision !== 0 && { aroundPrecision }),
```

Comment:
```suggestion
        ...(aroundPrecision && { aroundPrecision }),
```
Ça suffit normalement.
C'était pour être plus explicite ?

Reply:
Yes ca suffit - c'était pour être plus explicite mais ce n'est peut être pas nécessaire